addons/godot_rl_agents/sync.gd

```diff
@@ -2,165 +2,253 @@ extends Node
 
 # --fixed-fps 2000 --disable-render-loop
 
-enum ControlModes {HUMAN, TRAINING, ONNX_INFERENCE}
+enum ControlModes { HUMAN, TRAINING, ONNX_INFERENCE }
```

Comment:
Unnecessary spaces?

Reply:
Probably done by auto-formatting, can check by running it through again.

Reply:
What do you use to auto format your godot code?

Reply:
Previously I used the GDScript formatter VSCode extension on individual files I'm working on, but recently I started trying out the command line version, gdformat, with: https://github.com/Scony/godot-gdscript-toolkit, as I more frequently use the integrated Godot Editor gdscript IDE. I used the default line width, without providing any arguments for it.

In this commit on IL branch I reformatted some of the remaining files for that branch:
https://github.com/edbeeching/godot_rl_agents_plugin/pull/35/commits/da3f2bdf9590c85fae49017f8ef8746fab37e2ec

They do mention:
`Formatting may lead to data loss, so it's highly recommended to use it along with Version Control System (VCS) e.g. git`

So far it seems that the plugin is working after formatting, and the hope is to have a more consistent experience without having to worry as much about line length and other formatting during coding.